daemon/stats_unix_test.go

```diff
@@ -3,28 +3,154 @@
+cpu37 5 0 8 51289 0 0 1 0 0 0
+cpu38 3 0 12 51283 1 0 1 0 0 0
+cpu39 9 0 22 51274 0 0 1 0 0 0
+cpu40 7 0 17 51255 0 0 1 0 0 0
+cpu41 7 0 44 51238 0 0 1 0 0 0
+cpu42 1 0 8 51288 0 0 1 0 0 0
+cpu43 10 0 16 51268 3 0 1 0 0 0
+cpu44 7 0 12 51260 0 0 1 0 0 0
+cpu45 28 0 20 51253 0 0 1 0 0 0
+cpu46 8 0 7 51284 1 0 1 0 0 0
+cpu47 7 0 11 51286 0 0 1 0 0 0
+cpu48 2 0 16 51290 0 0 1 0 0 0
+cpu49 2 0 15 51284 1 0 1 0 0 0
+cpu50 8 0 13 51284 0 0 1 0 0 0
+cpu51 5 0 12 51280 2 0 1 0 0 0
+cpu52 8 0 13 51280 0 0 0 0 0 0
+cpu53 15 0 20 51262 0 0 1 0 0 0
+cpu54 4 0 12 51286 0 0 1 0 0 0
+cpu55 9 0 14 51275 2 0 1 0 0 0
+cpu56 5 0 17 51274 0 0 0 0 0 0
+cpu57 5 0 15 51278 1 0 0 0 0 0
+cpu58 6 0 16 51279 0 0 0 0 0 0
+cpu59 11 0 15 51267 0 0 0 0 0 0
+cpu60 5 0 6 51283 0 0 0 0 0 0
+cpu61 4 0 15 51275 0 0 0 0 0 0
+cpu62 7 0 11 51285 0 0 0 0 0 0
+cpu63 8 0 18 51280 0 0 0 0 0 0
+cpu64 10 0 24 51245 1 0 0 0 0 0
+cpu65 11 0 41 51189 1 0 0 0 0 0
+cpu66 16 0 25 51263 0 0 0 0 0 0
+cpu67 16 0 30 51251 0 0 0 0 0 0
+cpu68 5 0 10 51284 0 0 0 0 0 0
+cpu69 9 0 24 51270 2 0 0 0 0 0
+cpu70 6 0 15 51285 0 0 0 0 0 0
+cpu71 6 0 11 51283 0 0 0 0 0 0
+cpu72 11 0 16 51280 0 0 0 0 0 0
+cpu73 16 0 11 51274 0 0 0 0 0 0
+cpu74 7 0 10 51283 0 0 0 0 0 0
+cpu75 9 0 21 51266 0 0 0 0 0 0
+cpu76 4 0 7 51284 0 0 0 0 0 0
+cpu77 7 0 17 51275 0 0 0 0 0 0
+cpu78 3 0 11 51289 0 0 0 0 0 0
+cpu79 5 0 24 51273 0 0 0 0 0 0
+cpu80 0 0 16 51286 0 0 0 0 0 0
+cpu81 12 0 23 51268 0 0 0 0 0 0
+cpu82 4 0 125 51171 0 0 0 0 0 0
+cpu83 1 0 21 51278 0 0 0 0 0 0
+cpu84 6 0 14 51271 0 0 0 0 0 0
+cpu85 26 0 21 51256 0 0 0 0 0 0
+cpu86 9 0 7 51273 0 0 0 0 0 0
+cpu87 6 0 22 51274 0 0 0 0 0 0
+cpu88 4 0 4 51287 0 0 0 0 0 0
+cpu89 9 0 13 51275 0 0 0 0 0 0
+cpu90 20 0 26 51105 0 0 0 0 0 0
+cpu91 6 0 15 51275 0 0 0 0 0 0
+cpu92 3 0 12 51284 0 0 0 0 0 0
+cpu93 7 0 15 51267 0 0 0 0 0 0
+cpu94 10 0 12 51278 0 0 0 0 0 0
+cpu95 7 0 17 51279 0 0 0 0 0 0
+cpu96 1 0 13 51287 0 0 0 0 0 0
+cpu97 5 0 18 51277 0 0 0 0 0 0
+cpu98 2 0 7 51288 0 0 0 0 0 0
+cpu99 11 0 21 51260 0 0 0 0 0 0
+cpu100 2 0 13 51286 0 0 0 0 0 0
+cpu101 5 0 20 51275 0 0 0 0 0 0
+cpu102 0 0 13 51288 0 0 0 0 0 0
+cpu103 4 0 10 51283 0 0 0 0 0 0
+cpu104 3 0 15 51284 0 0 0 0 0 0
+cpu105 7 0 13 51283 0 0 0 0 0 0
+cpu106 1 0 12 51286 0 0 0 0 0 0
+cpu107 4 0 18 51278 0 0 0 0 0 0
+cpu108 9 0 7 51283 0 0 0 0 0 0
+cpu109 7 0 9 51281 0 0 0 0 0 0
+cpu110 8 0 9 51279 0 0 0 0 0 0
+cpu111 3 0 21 51278 0 0 0 0 0 0
+cpu112 9 0 10 51275 0 0 0 0 0 0
+cpu113 3 0 10 51287 0 0 0 0 0 0
+cpu114 5 0 11 51285 0 0 0 0 0 0
+cpu115 2 0 13 51284 0 0 0 0 0 0
+cpu116 9 0 11 51280 0 0 0 0 0 0
+cpu117 5 0 11 51285 0 0 0 0 0 0
+cpu118 4 0 12 51287 0 0 0 0 0 0
+cpu119 3 0 10 51285 0 0 0 0 0 0
+cpu120 5 0 13 51280 0 0 0 0 0 0
+cpu121 3 0 12 51288 0 0 0 0 0 0
+cpu122 3 0 7 51286 0 0 0 0 0 0
+cpu123 2 0 16 51281 0 0 0 0 0 0
+cpu124 2 0 8 51289 0 0 0 0 0 0
+cpu125 9 0 12 51274 0 0 0 0 0 0
+cpu126 3 0 8 51281 0 0 0 0 0 0
+cpu127 3 0 14 51286 0 0 0 0 0 0
+intr [ ... snipped for brevity ... ]
```

Comment:
The `intr` part was actually crucial here since the original change fixed a bug where too long line would make the function error out.

But yeah it might actually be too long for inlining...
So looks like we need to keep it in a separate file - although I'd be more happy to access it via `go:embed` instead of opening it at runtime.
This way the test is self-contained and verified at compile-time.

Reply:
>But yeah it might actually be too long for inlining...

I don't think its too long, actually. I just wasn't aware that the entire line was crucial.

I just restored the original /proc/stat output, and the tests are still passing.

Reply:
> I don't think its too long

Technically there's nothing wrong, but it makes reviewing awkward 🙈 

<img width="739" alt="image" src="https://github.com/user-attachments/assets/ed4a8c03-512e-4b45-8d3d-43f29f33d079" />

Reply:
Ah, yes. Ok, I've taken your suggested approach: restored `testdata/stat` and embedded the file contents so its not being read at runtime.